CONTRIBUTING.md

```diff
@@ -31,6 +31,26 @@ npm run test-only
 Note, this does not actually run a build, so you may have to call `npm run build` after
 performing any changes.
 
+### Running one test
+
+To run only one test of interest, you can add `"only": true` to `options.json` of any test.
+For illustration purposes, assume we would like to run the test [`test/fixtures/comments/basic/shebang-import`](https://github.com/babel/babylon/tree/7.0/test/fixtures/comments/basic/shebang-import).
```

Comment:
For example, let's say we want to only run the test for the [`test/fixtures/comments/basic/shebang-import`]() fixture.

Add the `only` option to its [`options.json`](https://github.com/babel/babylon/blob/7.0/test/fixtures/comments/basic/shebang-import/options.json):

```json
{
  "sourceType": "module",
  "only": true
}
```

Then, run the tests using the same command as before:

```sh
$ npm run test-only
  ```